server/app/services/barman-service.js

```diff
@@ -197,10 +267,90 @@ async function deleteBarmanById(barmanId) {
     return barman;
 }
 
+/**
+ * Get the services of a barman
+ *
+ * @param barmanId {number} barman id
+ * @param startDate {Date} starting date for services
+ * @param endDate {Date} ending date for services
+ * @returns {Promise<Array>} Barmen's services
+ */
+async function getBarmanServices(barmanId, startDate, endDate) {
+
+    logger.verbose('Barman service: retreive services of the barman ', barmanId );
+
+    const barman = await Barman.findById(barmanId);
+
+    if (!barman) throw createUserError('UnknownBarman', 'This Barman does not exist');
+    const Op = sequelize.Op;
```

Comment:
Don't import the operator from here, but directly from the `sequelize` package:
`const {Op} = require('sequelize')`